lua/hbac/setup.lua

```diff
@@ -8,6 +8,17 @@ local M = {
 		close_command = function(bufnr)
 			vim.api.nvim_buf_delete(bufnr, {})
 		end,
+		telescope = {
+			mappings = {
+				-- first key is for "normal", second for "insert" mode
+				close_unpinned = { "<C-,>", "<C-,>" },
+				delete_buffer = { "<C-.>", "<C-.>" },
+				toggle_all = { "<C-;>", "<C-;>" },
+				toggle_selections = { "<C-g>", "<C-g>" },
+			},
+			-- pinned icon, unpinned icon, pinned hl, unpinned hl
+			pin_icons = { "󰐃", "", "DiagnosticError", "Normal" },
+		},
```

Comment:
Was looking for non-conflicting default mappings but these are admittedly awkward. Could also just leave them blank and require a user to set them. Pin icons are nerd font icons.

I made all these positional to avoid extra keys and more nested tables. Please advise on these choices.

Reply:
You can also use the meta key (M) for mappings to avoid conflicts.